codestodians/food-basket-front-end
# [Chore: #163732261] Upgrade and update webpack and selected dependencies

#### What does this PR do?
- Updates webpack and dependencies that got affected by breaking changes

#### Description of Task to be completed?
- Update dependencies
- Refactor where necessary

#### How should this be manually tested?
- Check out into the `chore/update-and-upgrade-webpack-163732261` branch.
- Run `yarn`
- Run `yarn start` to ensure that the app starts properly.
- Run `yarn test` to ensure that tests run normally.

#### Any background context you want to provide?
- Updating webpack resulted in graphql and a couple of other dependencies needing fixes progressively. I admit I kept solving problems as they arose, not completely keeping track of every single error.
- Changed from the use of `import` syntax to commonjs since webpack is the bundler and the fact that the bundler is first getting transpiled might slow down things.
- There's work on the horizon regarding uglifying and minification, and we'd be looking to create further stories to make the app run faster.

#### What are the relevant pivotal tracker stories?
- [#163732261](https://www.pivotaltracker.com/story/show/163732261)
